commit 5d8bd37431d2047b88f7bbfe5150680956e10d1e (refs/changes/19/2019/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-06-01T19:19:19+02:00 (2 years, 4 months ago)
    
    WORKFLOW-213 workflow should update to node v4
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/etc/config.json.in b/etc/config.json.in
index ef2ed65..6345d80 100644
--- a/etc/config.json.in
+++ b/etc/config.json.in
@@ -68,11 +68,14 @@
             "modules": {
                 "urlModule": "url",
                 "http": "http",
-                "uuid": "node-uuid",
+                "uuid": "uuid",
                 "restify": "restify",
                 "async": "async",
                 "sdcClients": "sdc-clients",
-                "verror": "verror"
+                "vasync": "vasync",
+                "verror": "verror",
+                "imgmanifest": "imgmanifest",
+                "lstream": "lstream"
             },
             "ufdsUrl": "ldaps://{{{UFDS_SERVICE}}}",
             "ufdsDn": "{{{ufds_ldap_root_dn}}}",
diff --git a/package.json b/package.json
index 0df2396..6eb093d 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "workflow",
   "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "repository": {
@@ -10,15 +10,15 @@
   },
   "dependencies": {
     "wf-moray-backend": "git+https://github.com/joyent/node-workflow-moray-backend.git#c3504876",
-    "wf": "git+https://github.com/joyent/node-workflow.git#cac32eec0fa906adeba683f7f9436a9247740f11",
-    "bunyan": "1.3.4",
+    "wf": "git+https://github.com/joyent/node-workflow.git#aac927e4",
+    "bunyan": "1.8.1",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
-    "restify": "2.6.1",
-    "node-uuid": "1.4.0",
+    "restify": "4.1.1",
+    "uuid": "3.0.1",
     "async": "0.1.22",
-    "vasync": "1.6.1",
-    "sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#06e4b180ff74b708096e3c4449e628332eb75047",
-    "verror": "1.3.5",
+    "vasync": "1.6.4",
+    "sdc-clients": "10.2.0",
+    "verror": "1.6.0",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#e21ea53",
     "lstream": "0.0.4"
   },
diff --git a/sapi_manifests/workflow/template b/sapi_manifests/workflow/template
index 0635a2d..aa14f32 100644
--- a/sapi_manifests/workflow/template
+++ b/sapi_manifests/workflow/template
@@ -72,7 +72,7 @@
             "modules": {
                 "urlModule": "url",
                 "http": "http",
-                "uuid": "node-uuid",
+                "uuid": "uuid",
                 "restify": "restify",
                 "async": "async",
                 "sdcClients": "sdc-clients",
